data_structures/stacks/stack.py

```diff
@@ -110,7 +110,8 @@ def test_stack() -> None:
     assert not stack.is_empty()
     assert stack.is_full()
     assert str(stack) == str(list(range(10)))
-    assert stack.pop() == 9
+    test = stack.pop()
+    assert test == 9
```

Comment:
This is not required. Please revert it.

Reply:
I will, but Snyk claims this can have side effects

Reply:
Tools are here to help us, they're not 100% correct. We should include our human judgement to make a decision :)